ramls/edge-patron.raml

```diff
@@ -18,6 +18,7 @@ types:
   money: !include money.json
   item: !include item.json
   hold-cancellation: !include hold-cancellation.json
+    errors: !include raml-util/schemas/errors.schema
```

Comment:
Why is this definition indented further than the others?

Reply:
It's misprint I hope so. Will fix it.